bundles/org.openhab.binding.bluetooth.airthings/src/main/java/org/openhab/binding/bluetooth/airthings/internal/AbstractAirthingsHandler.java

```diff
@@ -111,23 +111,27 @@ private void executePeridioc() {
     }
 
     private synchronized void execute() {
-        ConnectionState connectionState = device.getConnectionState();
-        logger.debug("Device {} state is {}, serviceState {}, readState {}", address, connectionState, serviceState,
-                readState);
+        try {
```

Comment:
Following your (good) argument, the try/catch should be put in method executePeridioc, not method execute